assets/src/edit-story/components/dropTargets/provider.js

```diff
@@ -172,6 +172,23 @@ function DropTargetsProvider({ children }) {
         });
 
       setActiveDropTargetId(null);
+
+      // Auto-play video on drop
+      if (resource.type === 'video') {
+        // eslint-disable-next-line @wordpress/react-no-unsafe-timeout
+        setTimeout(() => {
+          const videoEl = document.getElementById(
+            `video-${activeDropTargetId}`
+          );
+          if (videoEl) {
+            videoEl.load();
+            const playPromise = videoEl.play();
+            if (playPromise) {
+              playPromise.catch(() => {});
+            }
+          }
```

Comment:
What do we do if there's no element? Try again in x millis? Ignore it? Explode?

Reply:
Not sure if we want some kind of retry mechanism here ATM, if that part will ever fail (bug report?) then we can do it.